pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.6.7</version>
+        <version>2.6.6</version>
```

Comment:
I tested with version `2.6.7` following thomas repository and works!

But reading the Spring documentation... I prefer to use the version `2.6.6`:

https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_validate_spring_boot_version
> Spring Native 0.11.4 only supports Spring Boot 2.6.6, so change the version if necessary.